xhr.bs

```diff
@@ -1792,7 +1792,8 @@ constructor must run these steps:
  <li><p>Let <var>fd</var> be a new {{FormData}} object.
 
  <li><p>If <var>form</var> is given, then set <var>fd</var>'s <a for=FormData>entry list</a> to the
- result of <a>constructing the form data set</a> for <var>form</var>.
+ result of <a lt="constructing the form data set">constructing the entry list</a> for
```

Comment:
Make this `<a>constructing the entry list</a>`. This'll fail until we've updated HTML with the new term and approximately one day has passed so the cross-specification indexes have had time to update. After that we can double check and it should be okay.

Reply:
Got it.  Thanks!

Reply:
@tkent-google I think this should instead use "selected files" and we should define that to be a list of `File` objects. WDYT?